CMakeLists.txt

```diff
@@ -8163,7 +8168,7 @@ IF(XNNPACK_BUILD_TESTS)
 
   ADD_EXECUTABLE(f16-dwconv-minmax-test test/f16-dwconv-minmax.cc)
   TARGET_INCLUDE_DIRECTORIES(f16-dwconv-minmax-test PRIVATE include src test)
-  TARGET_LINK_LIBRARIES(f16-dwconv-minmax-test PRIVATE cpuinfo fp16 pthreadpool gtest gtest_main)
+  TARGET_LINK_LIBRARIES(f16-dwconv-minmax-test PRIVATE cpuinfo fp16 pthreadpool gtest gtest_main logging)
```

Comment:
This line is for external dependencies. Internal dependencies are listed in alphabetical order on the line below. Ditto for other TARGET_LINK_LIBRARIES changes.

Reply:
Done